src/utils.h

```diff
@@ -160,80 +160,61 @@ std::string TdMeta(int prob, int td);
 std::string TdInput(int prob, int td);
 std::string TdOutput(int prob, int td);
 
-class fromVerdict {
-  const int verdict;
-
- public:
-  explicit fromVerdict(int verdict) : verdict(verdict) {}
-  const char* toStr() const {
-    switch (verdict) {
-      case AC:
-        return "Accepted";
-      case WA:
-        return "Wrong Answer";
-      case TLE:
-        return "Time Limit Exceeded";
-      case MLE:
-        return "Segmentation Fault";
-      case OLE:
-        return "Output Limit Exceeded";
-      case RE:
-        return "Runtime Error (exited with nonzero status)";
-      case SIG:
-        return "Runtime Error (exited with signal)";
-      case CE:
-        return "Compile Error";
-      case CO:
-        return "Compilation Timed Out";
-      case ER:
-        return "WTF!";
-      default:
-        return "nil";
-    }
-  }
-  std::string toAbr() const {
-    switch (verdict) {
-      case AC:
-        return "AC";
-      case WA:
-        return "WA";
-      case TLE:
-        return "TLE";
-      case MLE:
-        return "MLE";
-      case OLE:
-        return "OLE";
-      case RE:
-        return "RE";
-      case SIG:
-        return "SIG";
-      case CE:
-        return "CE";
-      case CO:
-        return "CO";
-      case ER:
-        return "ER";
-      default:
-        return "";
-    }
+inline const char* VerdictToStr(int verdict) {
```

Comment:
is it better to change `int verdict` to `RESULTS verdict`?

Reply:
Fixed at 1f73053.